### ISSUE TITLE
dragon is destroying chunk

### ISSUE THREAD

**Issue by OpsKern** [Alex-the-666/Ice_and_Fire#313]
not sure how it happened but this used to be a small cave system in a hill and after restarting the server, I came back to this.

![2017-09-12_14 01 27](https://user-images.githubusercontent.com/18408743/30341226-4230ff08-97c3-11e7-85e0-75c637710756.png)

and while typing this, he went back to sleep.

![2017-09-12_14 05 39](https://user-images.githubusercontent.com/18408743/30341286-7dfe8d7a-97c3-11e7-8cac-0f8b7421b69e.png)
 version 1.2.1 on 1.11.2 minecraft

**Comment by OpsKern** [comment]
there was a dragon roost/nest way below in the ground. Dragon had 496 hit points.
Here is a pic of the nest in relation to where the dragon came up.
![2017-09-12_16 35 58](https://user-images.githubusercontent.com/18408743/30346974-8c7e7f58-97d8-11e7-8363-672d4bf40dc0.png)

**Comment by OpsKern** [comment]
I've confirmed that when the dragon is awoken somehow, it flies around and breaks blocks. In this case, I'm guessing it broke into an area large enough to eventually start flying and made it's way up to the surface.

Maybe this isn't a bug but it may be a good idea to clear the dragon's aggro list unless the entity is a player. This guy has been awake for about 10 minutes now and he's already trashing the lair.

![2017-09-13_10 41 44](https://user-images.githubusercontent.com/18408743/30383711-4e3bce4c-9870-11e7-9585-919264ef78e7.png)

**Comment by OpsKern** [comment]
OK more information.

I used MCEdit to delete the chunks that had been destroyed and they regenerated just fine. A short while later, I was near the dragon's lair again, on the surface - see this screen shot and notice the location. 

![2017-09-13_13 18 39](https://user-images.githubusercontent.com/18408743/30391145-6930dbaa-9886-11e7-859f-a2c24a4b97fa.png)

The next screen shot is going to be further up by that yellow flower and looking down and to the right.

I attached a creeper at the location in the above screen shot and immediately, I was teleported into the dragon's lair and killed by it. You can see my death beacon in this next screen shot and the dead dragon - I did /kill @e quickly to prevent it from moving away

![2017-09-13_13 19 07](https://user-images.githubusercontent.com/18408743/30391256-b928f214-9886-11e7-8aa5-efe213c06566.png)

I will now use MCEdit to regenerate the chunks that are missing again. Let me know if there is anything other info I can give you to help.

**Comment by AlexModGuy** [comment]
They only wake if disturbed so a player must have happened across it and not killed it.

**Comment by Centros** [comment]
Ya I played 2 worlds and was wondering were these giant holes were coming from. I had an idea it was ice & fire but as I couldn't find anyone else talking about it I couldn't really confirm. Anywho on the second one I never even saw the dragon cave unless he has a huge agro field for players. I didn't even know it was there till I noticed on my map a huge hole and I went to investigate. Didn't see a dragon in the hole and I didn't hear him flying around at all during my play time. I am also the only one that has been on the server so far so the only player he could agro to would be me. Unless he can agro to other mobs or he has a like 2 or 3 chunk agro field I know I wasn't even close to the area he popped out of to wake him. Also side note love the mod I hope to see more but giant chunks taken out of the world suck.
